### PR TITLE
Make elevation plot settings global

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: common/src/main/scala/explore/common/UserPreferencesQueries.scala [hunk 1]
```diff
@@ -331,56 +329,41 @@ object UserPreferencesQueries {
 
   object ElevationPlotPreference:
     def updatePlotPreferences[F[_]: ApplicativeThrow](
-      uid:      User.Id,
-      targetId: Target.Id,
-      site:     Site,
-      range:    PlotRange,
-      time:     TimeDisplay
+      userId: User.Id,
+      site:   Site,
```
Comment (rpiaggio) [inline common/src/main/scala/explore/common/UserPreferencesQueries.scala:333]: Maybe we should keep the site bound to the target?

Comment (cquiroz) [reply common/src/main/scala/explore/common/UserPreferencesQueries.scala:333]: It was requested to make it global

**Review by rpiaggio — APPROVED** [review]
LGTM

**Comment by cquiroz** [comment]
After some discussion, the site won't be stored as a persistent setting

**Pull request merged**